src/index.js

```diff
@@ -1,9 +1,73 @@
 'use strict';
 
-const defaultOptions = {};
+const Matrix = require('ml-matrix');
+const Algebra = Matrix.algebra;
```

Comment:
@targos talking about that, when you import the ML.Matrix package from lactame, there's an Algebra object, do you know why?